scripts/diagnostic_gen.py

```diff
@@ -310,7 +316,7 @@ def createdocs(outDir, inpath, slangBin, diags, groups):
                 curropt[1] += " "
             curropt[1] += line
 
-    if curropt:
+    if len(curropt[0]) > 0:
```

Comment:
And this one?

Reply:
Consequence of the above. You could check for None before, but now it is assigned an array of None, so the original check would not work.

I'm not too fond of this change. I would like a solution upstream to avoid me maintaining a patch locally in our tree, so curious to hear whether you have a better way. But please also feel free to reject it if you "hate" it, and I'll deal with it.

Reply:
Same here.